package.json

```diff
@@ -14,6 +14,7 @@
     "refapp2Inpatient": "mvn test -Dcucumber.filter.tags='@selenium and @inpatient'",
     "refapp2Registration": "mvn test -Dcucumber.filter.tags='@selenium and @registration'",
     "refapp2FindPatient": "mvn test -Dcucumber.filter.tags='@selenium and @findPatient'",
+	"refapp2EditDemographics":"mvn test -Dcucumber.filter.tags='@selenium and @editDemographics'",
```

Comment:
On my pc this file shows it is well aligned until I push it to github. Can I do an edit right here on github and commit? @kdaud

Reply:
> Can I do an edit right here on github and commit?

yes